client/src/components/layouts/default.tsx

```diff
@@ -44,12 +44,14 @@ import Footer from '../Footer';
 import Header from '../Header';
 import OfflineWarning from '../OfflineWarning';
 import { Loader } from '../helpers';
+import envData from '../../../../config/env.json';
 
 // preload common fonts
 import './fonts.css';
 import './global.css';
 import './variables.css';
 import './rtl-layout.css';
+import StagingWarningModal from '../StagingWarningModal';
```

Comment:
This should be imported with the other modules, rather than the css. Next to signout-modal makes the most sense.

Reply:
done